### ISSUE TITLE
Use .cpp instead of .cc

### ISSUE THREAD

**Issue by jacobrosenthal** [ikeyasu/arduino-mock#22]
The makefile and associated arduino files are all cc, but Arduino uses .cpp so bringing in cpp files means renaming everything. it would be better to maintain their structure and naming as new files are brought in.

**Comment by ikeyasu** [comment]
Right. The reason why I have chosen a ".cc" is Google test uses it. It was misdirection, however.
